### PR TITLE
Remove Aeonik

### PR DESCRIPTION
**Pull request by ksalmon**

### CODE REVIEW

File: scss/_root.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-:root {
+:root, & {
```
Comment (arielkirkwood) [inline scss/_root.scss:1]: What's this?

**Review by arielkirkwood — CHANGES_REQUESTED** [review]
Could you rebase/reformat this PR/branch to not include built assets?

File: scss/_root.scss (hunk 1 above)
Comment (ksalmon) [reply scss/_root.scss:1]: I'll remove this

**Comment by ksalmon** [comment]
@arielkirkwood Why don't we want the current build in? Should I just add those directories/files to the gitignore?

**Comment by arielkirkwood** [comment]
@ksalmon I don't want to `.gitignore` them yet - but I'd prefer that we separate pull requests with "source code changes" from other commits or PRs that update all the "built"/`/dist` files.

**Pull request closed without merging**